unlock-app/src/components/interface/checkout/main/Metadata.tsx

```diff
@@ -187,7 +187,7 @@ export const MetadataInputs = ({
                   }}
                   ref={ref}
                   onBlur={onBlur}
-                  autoComplete={label}
+                  autoComplete="email"
```

Comment:
should take into account that this box can be an text input too. This doesn't currently. See above.

Reply:
good point!